### DISCUSSION TITLE
My first time

### DISCUSSION THREAD

**Discussion by Ayan-thecodeking** [github/india#598] [Students]
I'm applying for GitHub externship for very first time ... Any suggestions for me

**Comment by VinayAniruddh** [comment]
I'm applying for GitHub externship for very first time ... Any suggestions for me

**Comment by punitjain03** [comment]
Go through the organisation page and look at their website etc

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/ceMXzhfaka
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏
